packages/origin-247-certificate/src/offchain-certificate/synchronize/blockchain-synchronize.task.ts

```diff
@@ -0,0 +1,49 @@
+import { Injectable, Logger, OnModuleInit } from '@nestjs/common';
+import { InjectQueue, Process, Processor } from '@nestjs/bull';
+import { Queue } from 'bull';
+import { BlockchainSynchronizeService } from './blockchain-synchronize.service';
+
+const EVERY_15_MINUTES = '*/15  * * * *';
+const SYNCHRONIZATION_CONCURRENCY = 1;
+
+export const blockchainSynchronizeQueueName = 'blockchain-synchronize';
+
+@Injectable()
+@Processor(blockchainSynchronizeQueueName)
+export class BlockchainSynchronizeTask implements OnModuleInit {
```

Comment:
why this is using queue?

Reply:
Because I assumed we want to run it periodically. If we want to trigger that manually I would still leave it as queue to ensure there's only one synchronization task at the time.

Reply:
We should document this somewhere, probably in README or smth